generic/sysbench.py

```diff
@@ -67,6 +68,12 @@ def setUp(self):
                 self.bch = self.params.get('branch', default='master')
                 git.get_repo(self.urllink, branch=self.bch,
                              destination_dir=self.teststmpdir)
+                if self.urllink == 'https://github.com/akopytov/sysbench.git' \
+                        and "ppc" in distro.detect().arch:
```

Comment:
self.urllink == 'https://github.com/akopytov/sysbench.git'  this check is not required.

Reply:
@harihare i think it required as we pass different url from yaml file than it will fail , i already replied this on earlier :)